projects/aca-about/src/lib/about.component.ts

```diff
@@ -58,6 +61,7 @@ export class AboutComponent implements OnInit {
     if (this.authService.isEcmLoggedIn()) {
       this.setECMInfo();
     }
+    this.appServices.cast.subscribe((data) => (this.hideSidenav = data));
```

Comment:
this can be done by observable and async pipe, similar to extensions.... please check other files or read some docs on the async pipes.... You should also have "onDestroy" to properly unsubscribe... you can find plenty of examples in the existing code